server/game/Clocks/ChessClock.js

```diff
@@ -2,27 +2,68 @@ const Clock = require('./Clock');
 
 class ChessClock extends Clock {
     constructor(player, time) {
-        super(player, time);
+        super(player, time, 5);
         this.mode = 'stop';
         this.name = 'Chess Clock';
     }
 
+    pause() {
+        this.stop();
+    }
+
+    restart() {
+        this.start();
+    }
+
+    reset() {
+        this.stop();
+    }
+
```

Comment:
pause and reset both stop?

Reply:
Pause / Restart / Reset don't really mean anything with regards to clocks.  Chess clocks are either stopped (not counting down) or running (and counting down).  Adding a third state for "pausing" was meant to work with the manual mode commands, but didn't really do anything.  I added "manuallyPaused" to make the manual mode commands work, and so this became irrelevant.  I left them in because the other clocks potentially make better use of them, and I wanted to try to limit changes to the Chess Clocks as much as possible.